migrations/versions/8d260f51f8f1_update_decision_destinations.py

```diff
@@ -0,0 +1,62 @@
+"""update decision_destinations
+
+Revision ID: 8d260f51f8f1
+Revises: 2524def5b2d5
+Create Date: 2024-12-08 19:47:36.004281
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "8d260f51f8f1"
+down_revision = "2524def5b2d5"
+branch_labels = None
+depends_on = None
+
+
+# WARNING: This updates the pk of the decision_destinations table to include the id column
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("decision_destinations", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("id", sa.UUID(), nullable=False))
+        batch_op.drop_constraint(
+            "destionation_location_position_unique", type_="unique"
+        )
+        batch_op.create_unique_constraint(
+            "destination_location_position_unique",
+            ["decision_id", "destination_location_id", "position"],
+        )
+        # Drop the old primary key constraint
+        batch_op.drop_constraint("decision_destinations_pkey", type_="primary")
```

Comment:
To modify the PK need use this manually created Alembic statements. The SQLAlchemy migration couldn't autogenerate this itself.